modules/thrift/cmake/thrift.cmake

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Meta
+# SPDX-License-Identifier: Apache-2.0
+
+find_program(THRIFT_EXECUTABLE thrift)
+
+message(STATUS "Found thrift executable: ${THRIFT_EXECUTABLE}")
+
+function(thrift
```

Comment:
Pinging @tejlmand - should this be using the zephyr style of argument listing/usage/access e.g. `zephyr_file()` style or is this OK?

Reply:
If this were in `cmake/modules/extensions.cmake`, it might make sense to document it like other `zephyr_*()` functions but I very much think `extentions.cmake` would be the wrong place for it.

If there were a standard / conventional way to document cmake functions, I would be happy to use it, but there does not seem to be one.

Is the existing documentation not clear?